app/src/main/java/com/example/levi9application/common/Common.kt

```diff
@@ -15,6 +19,13 @@ import javax.inject.Singleton
 object Common {
     private const val BASE_URL = "https://www.thecocktaildb.com/api/json/v1/1/"
 
+    @Provides
+    fun provideContext(
+        @ApplicationContext context: Context
+    ): Context {
+        return context
+    }
+
     @Provides
     @Singleton
     fun provideAPI(): APIService {
```

Comment:
We should separately provide Retrofit client from the API interface. Separate these two into two methods.